src/main/resources/userEmailTemplate-feedbackSessionClosed.html

```diff
@@ -0,0 +1,13 @@
+Hello ${userName},
+
+The following feedback session is closed.
+<br>&nbsp;&nbsp; Course: [${courseId}]${courseName} 
+<br>&nbsp;&nbsp; Feedback Session Name: ${feedbackSessionName}
+<br>&nbsp;&nbsp; Deadline: ${deadline}
+<br>
+<p/>
+If you encounter any problems when using the system, you can email TEAMMATES support team at ${supportEmail}.
```

Comment:
`If you need any help or clarifications regarding TEAMMATES, please email us at ...`